docker-compose.yml

```diff
@@ -105,83 +105,83 @@ services:
       - net
     volumes:
       - ./configs/backend/:/etc/strelka/:ro
-    container_name: strelka_backend_1
+    container_name: sublime_strelka_backend_1
     depends_on:
-      - strelka-coordinator
-  strelka-manager:
+      - sublime_strelka_coordinator
+  sublime_strelka_manager:
     image: sublimesec/strelka-manager:0.3
     restart: unless-stopped
     command: strelka-manager
-    container_name: strelka_manager_1
+    container_name: sublime_strelka_manager_1
     networks:
       - net
     volumes:
       - ./configs/manager/:/etc/strelka/:ro
     depends_on:
-      - strelka-coordinator
-  strelka-coordinator:
+      - sublime_strelka_coordinator
+  sublime_strelka_coordinator:
     image: redis:alpine
     restart: unless-stopped
     command: redis-server --save "" --appendonly no
-    container_name: strelka_coordinator_1
+    container_name: sublime_strelka_coordinator_1
     networks:
       - net
-  screenshot-service:
+  sublime_screenshot_service:
     image: sublimesec/render-email-html:0.1
     restart: unless-stopped
     environment:
-      - S3_ENDPOINT=http://sublimes3:8110
+      - S3_ENDPOINT=http://sublime_s3:8110
       - SCREENSHOT_BUCKET=email-screenshots
       - AWS_REGION=us-east-1
       - DISABLE_DD=true
-    container_name: screenshot-service
+    container_name: sublime_screenshot_service
     env_file: sublime.env
     networks:
       - net
     depends_on:
-      - create-buckets
-  sublimes3:
-    container_name: sublimes3
+      - sublime_create_buckets
+  sublime_s3:
+    container_name: sublime_s3
     image: minio/minio
     restart: unless-stopped
     networks:
       - net
     volumes:
-      - s3_data:/data
+      - sublime_s3_data:/data
     env_file: sublime.env
     entrypoint: >
       /bin/sh -c "
       export MINIO_ROOT_USER=$$AWS_ACCESS_KEY_ID;
       export MINIO_ROOT_PASSWORD=$$AWS_SECRET_ACCESS_KEY;
       minio server --address 0.0.0.0:8110 --console-address 0.0.0.0:8111  /data;
       "
-  create-buckets:
+  sublime_create_buckets:
```

Comment:
@cameron-dunn-sublime this was the only container with something that kind of resembled an error:

```
2023-01-24 14:38:45 mc: <ERROR> Unable to initialize new alias from the provided credentials. Invalid Request (invalid hostname).
2023-01-24 14:38:45 Bucket created successfully `myminio/email-screenshots`.
2023-01-24 14:38:45 [2023-01-24 22:38:45 UTC] 4.0KiB email-screenshots/
```

Is this expected or do I need to update something here?